llvm/lib/Target/RISCV/RISCVInstrInfoF.td

```diff
@@ -388,11 +388,11 @@ foreach Ext = FExts in {
 } // foreach Ext = FExts
 
 let Predicates = [HasStdExtF], mayRaiseFPException = 0,
-    IsSignExtendingOpW = 1 in
+    IsSignExtendingOpW = 1, isMoveReg = 1 in
```

Comment:
I'm not sure what all isMoveReg is used for. On RV64, this instruction has sign extending behavior. It might not be safe to do generic optimizations with it.